test/wordy_webserver_response_test.js

```diff
@@ -0,0 +1,68 @@
+const assert = require('assert');
+const should = require('should');
+const sinon = require('sinon');
+const response = require('../src/wordy_webserver_response.js');
+
+describe('Wordy WebServer Response', () => {
+  var mock_dataStore, mock_registerUser, mock_InRequest, mock_OutRequest, webResponse, userId, errorId;
+
+  beforeEach(function(){
+    sandbox = sinon.sandbox.create();
+
+    mock_dataStore = {registerUser: () => {}};
+    mock_registerUser = sandbox.stub(mock_dataStore, 'registerUser');
+    mock_commandCallback = sandbox.spy();
+    mock_InRequest = {body: {command: '/wordy-in', user_id: userId}};
+    mock_OutRequest = {body: {command: '/wordy-out', user_id: userId}};
+    mock_UnknownRequest = {body: {command: '/wadus'}};
+    webResponse = new response.WebServerResponse(mock_dataStore);
+    userId = 'XXXX';
+    errorId = 'ERROR_XXXX';
+  });
+
+  afterEach(function(){
+    sandbox.restore();
+  });
+
+  it('Should display a nice homepage', () => {
+    should.equal(response.WebServerResponse.getHome(), 'WordyBot');
+  });
+
+  it('Should register a user', () => {
+    webResponse.processCommand(mock_InRequest, mock_commandCallback);
+
+    mock_registerUser.args[0][2]();
```

Comment:
`mock_registerUser.args[0][2]();` faking a positive Firebase response when trying to register a user. Tried to find a clearer way of reflecting / expressing this in the code, but haven't.

Some of the other tests using a similar trick have a comment, but I stopped at some point : /